client/mass/nav.js

```diff
@@ -31,6 +32,7 @@ headtip.d.style('z-index', 5555)
 // headtip must get a crazy high z-index so it can stay on top of all, no matter if server config has base_zindex or not
 
 // data elements for navigation header tabs
+const aboutTab = { top: 'ABOUT', mid: '', btm: '', subheader: 'about' }
```

Comment:
show word ABOUT in mid instead

Reply:
Maybe Home would be a better title for this tab if it is going to be the first one